src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -464,6 +465,13 @@ private void InitializeSuffixes()
             AddSuffix("MESSAGES", new NoArgsSuffix<MessageQueueStructure>(() => GetMessages()));
         }
 
+        public global::Part GetControlPart()
+        {
+            global::Part res = Vessel.GetReferenceTransformPart(); //this can actually be null if "control from here" has never been used
```

Comment:
I've tried a few tests, and I can't seem to find a way to get the control part to actually show as the root part.  You must have found an odd combination that caused the null reference.  I'm going to merge it and just edit the comment so that we don't imply that it will always be null if you don't call "control from here".